app/customapplications.rst

```diff
@@ -8,14 +10,66 @@ from source code.
 Please visit the following sites for working demo setups we have prepared
 for your convenience:
 
-- https://gitlab.com/appuio
-- https://bitbucket.org/appuio
-- https://github.com/appuio
-
-OpenShift Developer Guide
--------------------------
+- `GitLab.com/appuio <https://gitlab.com/appuio?filter=example>`__
+- `Bitbucket.org/appuio <https://bitbucket.org/appuio/?search=example>`__
+- `GitHub.com/appuio <https://github.com/appuio?q=example>`__
 
-Alternatively, you may consult the OpenShift Developer Guide on
-`Creating New Applications
-<https://docs.openshift.com/container-platform/3.11/dev_guide/application_lifecycle/new_app.html>`__,
+Alternatively, you may consult the `OpenShift Developer Guide
+<https://docs.openshift.com/container-platform/3.11/dev_guide/>`__
+on Creating new applications,
 which covers several languages, frameworks and strategies.
+
+Logging
+-------
+
+Every application should log to stdout. APPUiO will automatically collect
+that output and make it accessible via Kibana at `logging.appuio.ch
+<https://logging.appuio.ch>`__. See :ref:`application-logs` for more details.
+
+The Kibana user interface allows powerful queries on all captured application
+console output (you don't have access to cluster logs for security reasons),
+but it requires some data analyst thinking and style of working.
+
+Exception Tracking
+------------------
+
+For APM and exception tracking we suggest to use an external service like
+`Sentry`_, `New Relic`_ or `Datadog`_. (They all provide an API to automate
+configuring your integration, which we highly recommend doing!)
+They all allow to configure alerting or integrate with alert services.
+
+.. _Sentry: https://sentry.io/_/resources/customer-success/alert-rules/
+.. _New Relic: https://docs.newrelic.com/docs/alerts/new-relic-alerts/getting-started/introduction-new-relic-alerts
+.. _Datadog: https://docs.datadoghq.com/monitors/notifications/?tab=is_alert
+
+Monitoring & Alerting
+---------------------
+
+From a concept point of view, cloud applications should be *self-healing*
+and hence have less need to be monitored for everything. You should specify
+`resource requests and limits`_ for your Pods as well as `liveness and
+readiness probes`_, then APPUiO can automatically kill and restart your
+applications when they start using too much CPU or memory, or are stuck in
+a broken state. In addition, we suggest the following options to actively
+monitor your application and receive alerts:
+
+- **Prometheus (with AlertManager)**, self-hosted. It is best installed using
+  a `Helm chart`_. See the Prometheus docs on how to configure
+  `alert backends`_ and `alerting rules`_.
```

Comment:
The upstream Prometheus Helm Chart is not Openshift compatible, it doesn't make sense to link it.

Reply:
Interesting. That was exactly your proposal, though. So, which one should I link to instead?

Reply:
Yes, until @corvus-ch wrote in Chat that it doesn't work on Openshift.
I do not know. The current workaround seems to be to create a new repository out from a `helm template`, fix the incompatible stuff and add more RBAC.

Reply:
Maybe link the Prometheus operator?
It's not yet available on APPUiO for all users, but a customer could deploy his own operator for his own namespace...

Reply:
Sounds like, good that we talk about a problem that needs to be solved for our customers! :smirk:

Reply:
> Yes, until @corvus-ch wrote in Chat that it doesn't work on Openshift.

Seems like a problem that only affects parts of the helm chart, not the chart in general. https://github.com/helm/charts/issues/20797 (closed only due to inactivity)

Reply:
Discussed this internally with @corvus-ch and adapted the text accordingly. Until we can offer our customers a self-service solution we'll simply let them ask us for help.

This PR should now be ready for merging.